src/types.ts

```diff
@@ -258,5 +260,7 @@ export type Album = {
   handle: string
   artwork: string
   plays: number
+  favorites: number,
+  trackCount: number
```

Comment:
Nit. Prop names could be consistent. If we're using the Count suffix maybe update other vars too